src/metabase/models/data_model.clj

```diff
@@ -0,0 +1,39 @@
+(ns metabase.models.data-model
```

Comment:
we should have a better naming for this so it doesn't get confused with Toucan models?